package_linter.py

```diff
@@ -2660,6 +2660,13 @@ def helpers_sourcing_backuprestore(self):
                 'In the context of backup and restore scripts, you should load _common.sh with "source ../settings/scripts/_common.sh"'
             )
 
+    @test(only=["_common.sh"])
+    def no_progress_in_common(self):
+        if self.contains("ynh_script_progression"):
+            yield Error(
+                "You should not use `ynh_script_progression` in _common.sh because it will not work."
+            )
```

Comment:
```suggestion
            yield Info(
                "You should not use `ynh_script_progression` in _common.sh because it is likely to not work or be inconsistent."
            )
```

I would set this as an info only because probably some apps use this in the wild and we dont want them to drop to level 4 just because of this.

Reply:
I think your suggestion is unclear because it gives idea it might work when in fact it will 100% produce warning to end-user. I pushed a new rephrasing with new warning level, do you like it better?

Reply:
Yes I'm pretty sure it can work (though display funky progress bar), cf for example in https://github.com/YunoHost-Apps/languagetool_ynh/blob/master/scripts/_common.sh

Maybe it crashes in your case because of some other specificies, such as being the very first call, etc ..

Reply:
Sorry i was not clear. It did not crash but produced warnings. So it's not so bad but still ^^